src/types/integer.rs

```diff
@@ -252,44 +283,6 @@ fn big_integer_from_big_uint(value: BigUint) -> Int {
     Int::BigInt(BigInt::from(value))
 }
 
-impl TryFrom<&UInt> for i64 {
-    type Error = IonError;
-
-    fn try_from(value: &UInt) -> Result<Self, Self::Error> {
-        match value {
-            UInt::U64(uint) => i64::try_from(*uint).or_else(|_| {
-                decoding_error(format!(
-                    "Unsigned integer {uint:?} was too large to be represented as an i64."
-                ))
-            }),
-            UInt::BigUInt(big_uint) => i64::try_from(big_uint).or_else(|_| {
-                decoding_error(format!(
-                    "Unsigned integer {big_uint:?} was too large to be represented as an i64."
-                ))
-            }),
-        }
-    }
-}
-
-impl TryFrom<&UInt> for usize {
-    type Error = IonError;
-
-    fn try_from(value: &UInt) -> Result<Self, Self::Error> {
-        match value {
-            UInt::U64(uint) => usize::try_from(*uint).or_else(|_| {
-                decoding_error(format!(
-                    "Unsigned integer {uint:?} was too large to be represented as an usize."
-                ))
-            }),
-            UInt::BigUInt(big_uint) => usize::try_from(big_uint).or_else(|_| {
-                decoding_error(format!(
-                    "Unsigned integer {big_uint:?} was too large to be represented as an usize."
-                ))
-            }),
-        }
-    }
-}
-
```

Comment:
🗺️ These manual implementations of `TryFrom` have been supplanted by the macro we just saw above.